src/packs/ranged/Thermal Pistol.yml

```diff
@@ -1,16 +1,13 @@
 _id: OB1D4PoUmSrqiycU
 data:
-  ab: 0
+  ab: 1
   ammo:
     burst: false
     longReload: false
     max: 5
     suppress: false
     type: typeAPower
     value: 5
-  bundle:
-    amount: 1
-    bundled: false
   cost: 300
```

Comment:
Don't update or remove data unrelated to PR

Reply:
This one was handled in #34.